Configure

```diff
@@ -535,8 +535,6 @@ my @disable_cascades = (
 
     sub { $disabled{cmac}; } => [ "siv" ],
     "legacy"                 => [ "md2" ],
-
-    "cmp"               => [ "crmf" ],
```

Comment:
I am still a bit confued: doesn*t this revert a bit of #8897, is this correct?

Reply:
Ah, sorry, I confused that change with some other, older one. Fixed.